### PR TITLE
Add Statistics Per Cou

### PR DESCRIPTION
**Pull request by lionick**
Someone can see now statistics per cou.
If is coadmin, cmadmin can see the whole list of cous, if not can see those that is couadmin.

### CODE REVIEW

File: Controller/RciamStatsViewerServicesController.php [hunk 1]
```diff
@@ -52,17 +55,33 @@ public function index()
...
         $this->utils->getTotalLoginCounts($conn, 365)
       );
-      
+
       $vv_logincount_per_idp = ($this->utils->getLoginCountPerIdp($conn, 0)) ?: array();
       $vv_logincount_per_sp = ($this->utils->getLoginCountPerSp($conn, 0)) ?: array();
       $vv_logincount_per_month = ($this->utils->getLoginCountByRanges($conn)) ?: array();
+
+      $vv_status_enum[StatusEnum::Active] = 'Active';
+      /*$data['StatusEnum']['Approved'] = StatusEnum::Approved;
```
Comment (ioigoume) [inline Controller/RciamStatsViewerServicesController.php:64]: What are these?

File: Controller/RciamStatsViewerServicesController.php [hunk 2]
```diff
@@ -96,7 +116,7 @@ public function getdataforuserstiles()
   {
     $this->log(__METHOD__ . '::@', LOG_DEBUG);
     // We accept only Ajax Requests
-    if(!$this->request->is('Ajax')) {
+    if (!$this->request->is('Ajax')) {
```
Comment (ioigoume) [inline Controller/RciamStatsViewerServicesController.php:119]: Undo the white space import change here(https://spaces.at.internet2.edu/display/COmanage/COmanage+Coding+Style#COmanageCodingStyle-Alignment)
Also try to follow the same pattern in all other occasions.

File: Controller/RciamStatsViewerServicesController.php (hunk 1 above)
Comment (lionick) [reply Controller/RciamStatsViewerServicesController.php]: i removed the comments.

File: Controller/RciamStatsViewerServicesController.php (hunk 2 above)
Comment (lionick) [reply Controller/RciamStatsViewerServicesController.php]: ok done.

**Pull request closed without merging**